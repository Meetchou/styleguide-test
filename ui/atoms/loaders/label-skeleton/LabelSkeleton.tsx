import React, { FC } from 'react';

import Skeleton from '@ui/atoms/loaders/skeleton/Skeleton';

const LabelSkeleton:FC = () => (
  <Skeleton
    size='small'
    className='mt-1 mb-2 w-1/2 sm:w-1/4'
  />
);

export default LabelSkeleton;
