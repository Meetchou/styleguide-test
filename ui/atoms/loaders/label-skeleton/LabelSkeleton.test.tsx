import React from 'react';

import render from '@ui/testing/render';

import LabelSkeleton from './LabelSkeleton';

describe('ui/atoms/loaders/label-skeleton/LabelSkeleton', () => {
  const props = {};

  it('renders correctly', () => {
    const tree = render(<LabelSkeleton {...props} />);

    expect(tree.toJSON()).toMatchSnapshot();
  });
}); 
