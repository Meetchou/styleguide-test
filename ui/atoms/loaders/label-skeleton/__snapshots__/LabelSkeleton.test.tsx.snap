// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui/atoms/loaders/label-skeleton/LabelSkeleton renders correctly 1`] = `
<div
  className="animate-pulse bg-gray-100 rounded h-3 mt-1 mb-2 w-1/2 sm:w-1/4"
/>
`;
